src/components/layouts/PageMargins.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+
+const useStyles = createUseStyles({
+  root: {
+    paddingLeft: 16,
+    paddingRight: 16,
+  },
+});
+
+const PageMargins = ({ children }) => {
+  const classes = useStyles();
+  return <div className={classes.root}>{children}</div>;
+};
+
+export default PageMargins;
```

Comment:
I wanted a background color on half of the free trial page, but there was left & right padding built into the `LayoutControl` component which is used on basically every page of the site. I split out this padding functionality into this new component which can be optionally used on a case by case basis.